README.md

```diff
@@ -155,20 +155,34 @@ Type: `Boolean`
 
 If `true`, the modal will receive a `role` of `alertdialog`, instead of its default `dialog`.
 
+### defaultStyles
```

Comment:
How about `includeDefaultStyles`? I think `include` makes it clear that the value is a boolean.